### PR TITLE
EDSC-2547: Display native granule format on collections details page

### PR DESCRIPTION
**Pull request by macrouch**

### CODE REVIEW

File: static/src/js/components/CollectionDetails/CollectionDetailsBody.js [hunk 1]
```diff
@@ -57,6 +58,16 @@ const buildScienceKeywordList = (scienceKeywords) => {
   return scienceKeywordList
 }
 
+const buildNativeFormatList = (nativeFormats) => {
+  if (!nativeFormats.length) return []
```
Comment (abbottry) [inline static/src/js/components/CollectionDetails/CollectionDetailsBody.js]: Shouldn't this return a React fragment or null? One condition returns markup, the other returns a javascript array.

File: static/src/js/util/collectionMetadata/nativeFormat.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import { uniq } from 'lodash'
+
+/**
+ * Extracts the native format(s) from the UMM-C metadata
+ * @param {Object} ummJson UMM-C JSON metadata
+ */
+export const buildNativeFormat = (ummJson) => {
+  const { ArchiveAndDistributionInformation = {} } = ummJson
+  const { FileDistributionInformation = [] } = ArchiveAndDistributionInformation
```
Comment (abbottry) [inline static/src/js/util/collectionMetadata/nativeFormat.js]: Does this survive the linter? I thought it'd require the first letter to be lower case?

File: static/src/js/util/collectionMetadata/nativeFormat.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+  const formats = []
+
+  FileDistributionInformation.forEach((info) => {
+    const { Format, FormatType } = info
+    if (FormatType.toLowerCase() === 'native' && Format.toLowerCase() !== 'not provided') formats.push(Format)
```
Comment (abbottry) [inline static/src/js/util/collectionMetadata/nativeFormat.js]: Can this please be three lines, this is much less readable (IMO)

File: static/src/js/components/CollectionDetails/CollectionDetailsBody.js [hunk 4]
```diff
@@ -11,6 +11,7 @@ import CollectionDetailsMinimap from './CollectionDetailsMinimap'
 import SplitBadge from '../SplitBadge/SplitBadge'
 
 import './CollectionDetailsBody.scss'
+import pluralize from '../../util/pluralize'
```
[the remark this answers is not in the document]
Comment (trevorlang) [reply static/src/js/components/CollectionDetails/CollectionDetailsBody.js]: +1. I think we should be using named imports everywhere except for the react components

File: static/src/js/components/CollectionDetails/CollectionDetailsBody.js (hunk 1 above)
Comment (macrouch) [reply static/src/js/components/CollectionDetails/CollectionDetailsBody.js]: I just did the same thing that buildRelatedUrlsList and buildScienceKeywordList are doing

Comment (abbottry) [reply static/src/js/components/CollectionDetails/CollectionDetailsBody.js]: ok -- well then those are wrong too! I think the 'issue' is that we always have a conditional that wraps the render when these are used so it never actually attempts to render the array. @trevorlang any idea how to best handle this? I know it hasn't caused issues (that I'm aware of) but definitely seems like a quick and easy improvement.

**Pull request merged**